packages/core/src/callbacks/SilentCallback.component.tsx

```diff
@@ -1,7 +1,12 @@
-import React, { useEffect } from 'react';
+import React, { FC, useEffect } from 'react';
 import { UserManager } from 'oidc-client';
+import { oidcLog } from '../services';
 
-const SilentCallback = ({ logger }) => {
+interface SilentCallbackProps {
```

Comment:
Do you Know why are you using interface and not type in that case?

Reply:
Good question: i was not sure about using interface or type. Your tslint settings throws error `interface-over-type-literal  Use an interface instead of a type literal.`
But sometimes i'm not sure we want to provide ['extensible' type](https://github.com/typescript-cheatsheets/react-typescript-cheatsheet#types-or-interfaces). Do you have an opinion about this?